first_django_pr/my_app/views.py

```diff
@@ -33,15 +33,12 @@
 def index(request):
     return HttpResponse(index_template)
 
-#Дописав
 def login(request):
     if request.method == 'POST':
         name = request.POST.get('name')
         password = request.POST.get('password')
         return HttpResponse(f"Username: {name}, password: {password}")
```

Comment:
During login view, you do not create any data in the memory or in the database it would be great if you could add that step as well

Reply:
I didn't manage to add it, I leave it like that for now

Reply:
If you are expecting that your other views should work with the existing User with the provided `name` and `password` it is the view where some model or class initialization should be implemented

Reply:
I wanted to make a username and password just for example. The password can always be replaced with @mail or Nickname or something else, but it seemed to me not so interesting. This is a one-time case, just an example.